packaging/linux/debian/control

```diff
@@ -10,7 +10,7 @@ Homepage: http://www.get-notes.com/
 
 Package: notes
 Architecture: amd64
-Depends: libqt5widgets5 (>= 5.2), libqt5gui5 (>= 5.2), libqt5network5 (>= 5.2), libqt5sql5 (>= 5.2), libqt5core5a (>= 5.2), sqlite3, ${misc:Depends}, ${shlibs:Depends}
+Depends: libqt5widgets5 (>= 5.2), libqt5gui5 (>= 5.2), libqt5network5 (>= 5.2), libqt5sql5 (>= 5.2), libqt5core5a (>= 5.2), sqlite3, libqt5sql5-sqlite (>= 5.2), ${misc:Depends}, ${shlibs:Depends}
```

Comment:
Nitpick: This is not a blocker, but after PR #390, Notes will not compile on any Qt version older than 5.9.5.